<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GimbalRemote</class>
 <widget class="QWidget" name="GimbalRemote">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1028</width>
    <height>558</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>441</width>
     <height>251</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <widget class="QPushButton" name="rollUp">
        <property name="text">
         <string>Roll Up</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="rollDown">
        <property name="text">
         <string>Roll Down</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="2" column="0">
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QPushButton" name="yawUp">
        <property name="text">
         <string>Yaw Up</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="yawDown">
        <property name="text">
         <string>Yaw Down</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QPushButton" name="pitchUp">
        <property name="text">
         <string>Pitch Up</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pitchDown">
        <property name="text">
         <string>Pitch Down</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="2" column="1">
     <widget class="QSlider" name="sliderYaw">
      <property name="minimum">
       <number>-1000</number>
      </property>
      <property name="maximum">
       <number>1000</number>
      </property>
      <property name="singleStep">
       <number>1</number>
      </property>
      <property name="orientation">
       <enum>Qt::Orientation::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QSlider" name="sliderPitch">
      <property name="minimum">
       <number>-1000</number>
      </property>
      <property name="maximum">
       <number>1000</number>
      </property>
      <property name="singleStep">
       <number>1</number>
      </property>
      <property name="orientation">
       <enum>Qt::Orientation::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QSlider" name="sliderRoll">
      <property name="minimum">
       <number>-1000</number>
      </property>
      <property name="maximum">
       <number>1000</number>
      </property>
      <property name="orientation">
       <enum>Qt::Orientation::Horizontal</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>380</y>
     <width>411</width>
     <height>151</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_7">
      <item>
       <widget class="QLineEdit" name="relaRoll"/>
      </item>
      <item>
       <widget class="QLineEdit" name="relaPitch"/>
      </item>
      <item>
       <widget class="QLineEdit" name="relaYaw"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLineEdit" name="absoRoll"/>
      </item>
      <item>
       <widget class="QLineEdit" name="absoPitch"/>
      </item>
      <item>
       <widget class="QLineEdit" name="absoYaw"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_8">
      <item>
       <widget class="QLineEdit" name="absoVelRoll"/>
      </item>
      <item>
       <widget class="QLineEdit" name="absoVelPitch"/>
      </item>
      <item>
       <widget class="QLineEdit" name="absoVelYaw"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QPlainTextEdit" name="plainTextEdit">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>340</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="plainText">
    <string>Relative</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="plainTextEdit_2">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>340</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LayoutDirection::LeftToRight</enum>
   </property>
   <property name="plainText">
    <string>Absolute</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="plainTextEdit_3">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>340</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="plainText">
    <string>Velocity</string>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>40</y>
     <width>381</width>
     <height>101</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPushButton" name="zoomIn">
        <property name="text">
         <string>Zoom In</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="zoomOut">
        <property name="text">
         <string>Zoom Out</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_9">
      <item>
       <widget class="QPushButton" name="btnNeutral">
        <property name="text">
         <string>Neutral</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnFocus">
        <property name="text">
         <string>Focus</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_10">
      <item>
       <widget class="QPushButton" name="goTo">
        <property name="text">
         <string>Go to Point A</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="goToB">
        <property name="text">
         <string>Go to Point B</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>180</y>
     <width>221</width>
     <height>181</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_12">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_11">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <item>
           <widget class="QPushButton" name="pushButton">
            <property name="text">
             <string>Target Detection</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnTrack">
            <property name="text">
             <string>Track</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="1" column="1">
           <widget class="QLineEdit" name="editLastY"/>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="editLastX">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="contextMenuPolicy">
             <enum>Qt::ContextMenuPolicy::DefaultContextMenu</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="ctaProtocol">
        <property name="text">
         <string>Click to aim (Protocol)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="ctaCustom">
        <property name="text">
         <string>Click to aim (Custom)</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
